src/scripts.js

```diff
@@ -1,8 +1,6 @@
 // import MicroModal from 'micromodal';
 // var MicroModal = require('../node_modules/micromodal');
-// const ingredientsData = require('../data/ingredients')
-// const recipeData = require('../data/recipes')
-// const usersData = require('../data/users')
+// const RecipeRepository = require('./RecipeRepository');
```

Comment:
not sure I should be pushing these changes to script.js and the HTML file. might have some wild merge conflicts to deal with later.